hooks/pre-commit

```diff
@@ -13,15 +13,16 @@ readonly green=$(tput bold; tput setaf 2)
 
 exit_code=0
 
-echo -ne "Checking that it builds..."
-if ! OUT=$(make 2>&1); then
-    echo
-    echo "${red}${OUT}"
-    exit_code=1
-else
-    echo "${green}OK"
-fi
-echo "${reset}"
+# comment it by default. You can uncomment it.
```

Comment:
@coocood complains about building is too slow. And sometime, we have to commit even our changes break build.